src/RandomQuantum.jl

```diff
@@ -130,10 +130,10 @@ end
 
 function rand(dist::ClosedHaarEnsemble)
     X = rand(GinibreEnsemble(dist.dim))
-    Q,_ = qr(X)
+    Q = qr(X).Q * I # yields dense square matrix
```

Comment:
I think you must have `I` defined somewhere in your code. I think this looks good to me other than maybe a change like this:
```suggestion
    Q = qr(X).Q * eye(dist.dim) # yields dense square matrix
```

Reply:
`I` is exported from `LinearAlgebra`, so should be available, in contrast to `eye`, which doesn't exist in Base or `LinearAlgebra`.

Reply:
Aha, I see, `eye` is something imported in this package.